test/fixtures/desks.yml

```diff
@@ -8,3 +8,8 @@ desk1:
   name: desk1
   grouping: grouping1
   occupied: true
+
+desk2:
```

Comment:
Since you're already here, why don't we give these fixtures a more descriptive name? `desk1` and `desk2` don't tell me anything about the desk itself